### PR TITLE
Fix object order after select.

### PR DESCRIPTION
**Pull request by Anakonda**
Fixes Opengamebox2/opengamebox2#30

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -79,24 +79,34 @@ io.on('connection', socket => {
...
-            selectedClientId: entity.selectedClientId,
-            depth: entity.depth,
+            selectedClientId: null,
+            depth: entities[entity.id].depth,
           });
         }
       }
     });
+    entityArr.forEach((entity, index, array) => {
+      if (entities[entity.id].selectedClientId != null
```
Comment (vileanco) [inline src/index.js]: !==

File: src/index.js [hunk 2]
```diff
@@ -79,24 +79,34 @@ io.on('connection', socket => {
...
+        && entities[entity.id].selectedClientId != player.id) {
```
Comment (vileanco) [inline src/index.js]: !==

File: src/index.js [hunk 3]
```diff
@@ -79,24 +79,34 @@ io.on('connection', socket => {
...
+        return
```
Comment (vileanco) [inline src/index.js]: ;

File: src/index.js [hunk 4]
```diff
@@ -79,24 +79,34 @@ io.on('connection', socket => {
...
+      entities[entity.id].depth = ++nextEntityDepth;
+      updateList.push({
+        id: entity.id,
+        selectedClientId: entities[entity.id].selectedClientId,
+        depth: entities[entity.id].depth,
+      });
+    });
+
+
+
```
Comment (vileanco) [inline src/index.js]: Too much whitespace

File: src/index.js [hunk 5]
```diff
@@ -79,24 +79,34 @@ io.on('connection', socket => {
   onRequest(socket, protocol.requests.ENTITY_SELECT_REQUEST, (entityArr, player) => {
     const updateList = [];
```
Comment (Ape) [inline src/index.js]: Add new line here.

**Pull request merged**